### PR TITLE
Support replacable arguments in command as well (besides arguments) in container op.

### PR DESCRIPTION
**Pull request by qimingj**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/623)

### CODE REVIEW

File: sdk/python/kfp/dsl/_container_op.py [hunk 1]
```diff
@@ -63,24 +59,25 @@ def __init__(self, name: str, image: str, command: str=None, arguments: str=None
...
+    to_search = []
     if arguments:
-      for arg in arguments:
-        match = re.findall(r'{{pipelineparam:op=([\w-]*);name=([\w-]+);value=(.*?)}}', str(arg))
-        matches += match
+      to_search += arguments
+    if command:
+      to_search += command
+
+    for arg in to_search:
```
Comment (Ark-kun) [inline sdk/python/kfp/dsl/_container_op.py]: for arg in (command or []) + (arguments or []):

Comment (qimingj) [reply sdk/python/kfp/dsl/_container_op.py]: Good idea. PTAL

**Pull request merged**